recipes/config_node.rb

```diff
@@ -80,4 +80,8 @@
   requires_authbind requires_authbind
   action :add_to_services
   not_if "systemctl is-active --quiet #{service_name}"
+end
+
+hopsworks_configure_server "generate-int-api-key" do
+  action :generate_internal_api_key
```

Comment:
@kouzant - Don't we need this in the das node as well?

Reply:
Actually `hopsworks::default` is sufficient both for HA and non-HA. In both configurations we deploy Hopsworks apps in default recipe